x/auth/client/testutil/suite.go

```diff
@@ -1460,6 +1464,299 @@ func (s *IntegrationTestSuite) TestSignWithMultiSignersAminoJSON() {
 	require.Equal(sdk.NewCoins(val0Coin, val1Coin), queryRes.Balances)
 }
 
+func (s *IntegrationTestSuite) TestTipsWithFee() {
+	require := s.Require()
+	val := s.network.Validators[0]
+	val0Coin := sdk.NewCoin(fmt.Sprintf("%stoken", val.Moniker), sdk.NewInt(10))
+
+	testCases := []struct {
+		name      string
+		args      []string
+		expectErr bool
+	}{
+		{
+			"error with SIGN_MODE_DIRECT_AUX and --aux unset",
+			[]string{
+				fmt.Sprintf("--%s=%s", flags.FlagSignMode, flags.SignModeDirectAux),
+			},
+			true,
+		},
+		{
+			"no error with SIGN_MDOE_DIRECT_AUX mode and generate-only set (ignores generate-only)",
+			[]string{
+				fmt.Sprintf("--%s=%s", flags.FlagSignMode, flags.SignModeDirectAux),
+				fmt.Sprintf("--%s=true", flags.FlagGenerateOnly),
+			},
+			false,
+		},
+		{
+			"no error with SIGN_MDOE_DIRECT_AUX mode and generate-only, tip flag set",
+			[]string{
+				fmt.Sprintf("--%s=%s", flags.FlagSignMode, flags.SignModeDirectAux),
+				fmt.Sprintf("--%s=true", flags.FlagGenerateOnly),
+				fmt.Sprintf("--%s=%s", flags.FlagTip, val0Coin.String()),
+			},
+			false,
+		},
```

Comment:
could we add 2 more tests:

- error with --aux --tip set, but --tipper unset
- no error with --aux --tip --tipper set

Reply:
> no error with --aux --tip --tipper set

this is already covered in happy case